packages/manager/modules/octavia-load-balancer/src/loadbalancers/loadbalancers.routing.js

```diff
@@ -0,0 +1,85 @@
+import { TRACKING_OCTAVIA_LOAD_BALANCERS_PREFIX } from '../octavia-load-balancer.constants';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('octavia-load-balancer.loadbalancers', {
+    url: '/loadbalancers?loadBalancerDeleteSuccess',
+    component: 'octaviaLoadBalancerListing',
+    redirectTo: (transition) =>
+      transition
+        .injector()
+        .getAsync('resources')
+        .then((resources) =>
+          resources.length === 0 ? 'octavia-load-balancer.onboarding' : false,
+        ),
+    resolve: {
+      projectId: /* @ngInject */ ($transition$) =>
```

Comment:
No need to add projectId and project as they already exist on parent state

Reply:
Done